src/components/Cateogory/Cateogory.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+export const Cateogory = () => {
+    const [error, setError] = useState();
+    const [cateogory, setCateogory] = useState();
+
+    useEffect(()=>{
+        (async ()=>{
+            try{
+                await axios.get("api/categories")
+                .then(({ data : {categories} })=>{
```

Comment:
No need to use .then if you are using async-await. You can wrap it in try catch block to handle error, and directly use the response object you get from the await call.